rasa/core/policies/memoization.py

```diff
@@ -239,20 +251,52 @@ def predict_action_probabilities(
         return self._prediction(result)
 
     def _metadata(self) -> Dict[Text, Any]:
-        return {
-            "priority": self.priority,
-            "max_history": self.max_history,
-            "lookup": self.lookup,
-        }
+        return {"lookup": self.lookup}
```

Comment:
How about dumping `_metadata` from the superclass as it's not overridden by anything?

Reply:
It's referenced in the persist method. I think it would be good to keep it for the sake of describing the interface too

Reply:
but other policies are overriding that anyway. How about moving it from `Policy` then to `MemoizationPolicy`? Especially since it's not really part of the interface anymore (all the policies have to call `persist` themselves anyway)

Reply:
ok moved the specific implementation

Reply:
Hm, not sure it was really a good thing. I think after all it's a decent default implementation and gives some useful structure to future policies, even though it might not be used by more than 1 policy rn

Reply:
I see your point. Just wondering if it actually provides a useful structure as they still have to remember to call it / some policies might not need persistence anyway

Reply:
alright I removed those in the PolicyGraphComponent